client/public/frame.html

```diff
@@ -18,23 +18,24 @@
   </script>
   <!--
     JSPM Generator Import Map
-    Edit URL: https://generator.jspm.io/#Y2NnYGCzD80rySzJSU1hSMpM183MK0lNTy1yMNQz0zM1ZEhJTc7MTczRyyp2MDTQM9YzZChKTUwu0U3Jz3UwNNMzxCqiX5xaVJZaBJGAKystzUxxsACaYQQAoBlP83cA
+    Edit URL: https://generator.jspm.io/#Y2VhYGCzD80rySzJSU1hSMpM183MK0lNTy1yMNQz0zM1ZEhJTc7MTczRyyp2MDTQM9YzZChKTUwu0U3Jz3UwNNMzxCqiX5xaVJZaBJGAKystzUxxsACaYQQAQSdSGncA
   -->
   <script type="importmap">
   {
     "imports": {
       "big-integer": "https://ga.jspm.io/npm:big-integer@1.6.51/BigInteger.js",
       "decimal.js": "https://ga.jspm.io/npm:decimal.js@10.3.1/decimal.js",
-      "react": "https://ga.jspm.io/npm:react@16.13.1/index.js",
-      "react-dom": "https://ga.jspm.io/npm:react-dom@16.13.1/index.js",
-      "react-dom/server": "https://ga.jspm.io/npm:react-dom@16.13.1/server.browser.js",
+      "react": "https://ga.jspm.io/npm:react@16.13.1/dev.index.js",
+      "react-dom": "https://ga.jspm.io/npm:react-dom@16.13.1/dev.index.js",
+      "react-dom/server": "https://ga.jspm.io/npm:react-dom@16.13.1/dev.server.browser.js",
```

Comment:
I wonder, does bumping the version also fix this? I would think this might need to be in sync with the `purescript-react` libraries which were recently updated to 18 (I'm actually surprised that `createRoot` is in the `dev` version of `16.13.1`).

Reply:
Looks like you're right. Bumping the React version to `17.0.2`, the last release before 18, makes it work.